crates/store/re_log_types/src/index/timestamp_format.rs

```diff
@@ -0,0 +1,31 @@
+/// How to display a [`crate::Timestamp`].
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+pub enum TimestampFormat {
+    /// Convert to local timezone and display as such.
+    LocalTimezone,
+
+    /// Display as UTC.
+    Utc,
+
+    /// Show as seconds since unix epoch
+    UnixEpoch,
+}
+
+impl TimestampFormat {
+    pub fn to_jiff_tz(self) -> jiff::tz::TimeZone {
```

Comment:
`tz` is a bit too unclear without having lots of context imho
```suggestion
    pub fn to_jiff_time_zone(self) -> jiff::tz::TimeZone {
```